pallets/posts/tests/src/mock.rs

```diff
@@ -150,9 +149,33 @@ impl pallet_space_follows::Config for Test {
     type WeightInfo = ();
 }
 
+parameter_types! {
+    pub const MaxDomainLength: u32 = 64;
+}
+pub struct MockEmptyDomainsProvider;
+impl DomainsProvider<AccountId> for MockEmptyDomainsProvider {
```

Comment:
Why DomainsProvider in POSTS?

Reply:
It's for `DomainsProvider` trait implementation. This trait is quite tricky, so it's better to implement it on-demand. In this case we don't actually need its functionality, just mocking "Ok" results.